scripts/db-clone/clone.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+
+set -x
+
+
+export PGHOST=pathoplexus3.cluster-che6ccwa8oie.eu-central-1.rds.amazonaws.com
+export PGUSER=postgres
```

Comment:
For reading the production database, could we create a read-only user? postgres is really powerful and we should avoid using it as much as possible.